src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Utilities.cs

```diff
@@ -48,6 +48,14 @@ public static void ExtractTarball(string tarballPath, string outputDir, string t
     }
 
     public static IEnumerable<string> GetTarballContentNames(string tarballPath)
+    {
```

Comment:
This could be simplified.

public static IEnumerable<string> GetTarballContentNames(string tarballPath) =>
    GetTarballContent(tarballPath).Select(entry => entry.Name);

Reply:
Addressed [here](https://github.com/dotnet/installer/pull/17233/files#diff-3e52a7713dbdb53496cf7dc21844b2eb5e5934d94c9cd8f54d30106af2f0d21fR51)